### DISCUSSION TITLE
使用antd-mobile@5.15.0版本组件报错

### DISCUSSION THREAD

**Discussion by liujian18720923143** [ant-design/ant-design-mobile#5349] [Q&A] [answered]
<img width="479" alt="企业微信截图_16555354931257" src="https://user-images.githubusercontent.com/52194781/174426786-da229628-c2b4-4bad-87f3-2fdc13e48411.png">

./node_modules/@floating-ui/dom/dist/floating-ui.dom.esm.js
Module parse failed: Unexpected token (305:11)
You may need an appropriate loader to handle this file type.
|   }
|
|   return { ...rect,
|     x: rect.x - scroll.scrollLeft + offsets.x,
|     y: rect.y - scroll.scrollTop + offsets.y

我调用antd-mobile组件时报错

**Comment by GOWxx** [comment, marked as answer]
可以提供一个复现demo，可以是 [codesandbox](https://codesandbox.io/s/antd-mobile-snrxr) / [stackblitz](https://stackblitz.com/edit/antd-mobile?file=index.tsx) / github repo / 压缩包 等等，这样可以更好的帮助到你

**Reply by liujian18720923143** [reply]
http://localhost:8005/ebsmobile/#/user/AA/AA/route/AAAA0000

**Reply by GOWxx** [reply]
> 代码ya'suo'bao压缩包

你这个压缩包可以删掉重新搞一个吗？
第一个原因是有太多你们自己项目的信息，甚至还有 .git 相关的信息，请注意安全
第二个原因是复现 demo 通常要求是最小复现，这里有太多无关的内容

**Reply by liujian18720923143** [reply]
好的谢谢

**Reply by liujian18720923143** [reply]
![image](https://user-images.githubusercontent.com/52194781/174433066-be945436-05a6-4ddd-bbf7-5a37c631468f.png)
现在不会报错但是只要把form放开就会报错
[UI_mobile.zip](https://github.com/ant-design/ant-design-mobile/files/8933339/UI_mobile.zip)

**Reply by GOWxx** [reply]
> ![image](https://user-images.githubusercontent.com/52194781/174433066-be945436-05a6-4ddd-bbf7-5a37c631468f.png) 现在不会报错但是只要把form放开就会报错 [UI_mobile.zip](https://github.com/ant-design/ant-design-mobile/files/8933339/UI_mobile.zip)

尝试配置一下 babel ？
我看它报错可能是不支持 es6 语法
可以配置整个 antd-mobile 转 es5 试试？

**Reply by liujian18720923143** [reply]
你试过么？
它是node_modules的包报错

**Reply by liujian18720923143** [reply]
而且我不会roadhog配置

**Reply by GOWxx** [reply]
在 roadhog 的配置文件 `.webpackrc.js` 内新增配置项：
  `extraBabelIncludes: ['./node_modules/antd-mobile', './node_modules/@floating-ui'],`

![image](https://user-images.githubusercontent.com/12870715/174513109-ed2b8019-6642-472b-9a20-f3f64426cc27.png)

配置文档参考：
https://github.com/sorrycc/roadhog

我也不会 roadhog 配置，也是临时去查找去看的

**Reply by liujian18720923143** [reply]
谢谢，已经可以了，我之前也看过这个属性但是我没有看懂怎么用。所以一直报错，跟webpack相差太大了，之前webpack还需要babel的。真的是非常感谢！

**Comment by liujian18720923143** [comment]
能加个微信或者qq吗？方便交流




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年6月20日(星期一) 上午10:10
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ant-design/ant-design-mobile] ***@***.***版本组件报错 (Discussion #5349)





 
在 roadhog 的配置文件 .webpackrc.js 内新增配置项：
 extraBabelIncludes: ['./node_modules/antd-mobile', ***@***.***'],
 

 
配置文档参考：
 https://github.com/sorrycc/roadhog
 
我也不会 roadhog 配置，也是临时去查找去看的

**Reply by GOWxx** [reply]
欢迎加入antd-mobile 社区[钉钉交流群](https://mobile.ant.design/zh/guide/quick-start#%E8%AE%A8%E8%AE%BA%E7%BB%84)

**Comment by liujian18720923143** [comment]
好的谢谢



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年6月20日(星期一) 中午11:12
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ant-design/ant-design-mobile] ***@***.***版本组件报错 (Discussion #5349)





   
欢迎加入antd-mobile 社区钉钉交流群
